src/components/RequiredCountrySelect.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useState } from 'react';
 import { Country } from '../services/api-types';
-import { CountrySelect } from './CountrySelect';
+import { PlaceSelect } from './PlaceSelect';
```

Comment:
Since you renamed `CountrySelect` to `PlaceSelect`, you should also rename `RequiredCountrySelect` to `RequiredPlaceSelect`.